src/components/info-window.tsx

```diff
@@ -176,6 +177,47 @@ export const InfoWindow = (props: PropsWithChildren<InfoWindowProps>) => {
     const openOptions: google.maps.InfoWindowOpenOptions = {map};
     if (anchor) {
       openOptions.anchor = anchor;
+
+      // Only do the infowindow adjusting when dealing with an AdvancedMarker
+      if (isAdvancedMarker(anchor) && anchor.content instanceof Element) {
+        const {width: anchorWidth, height: anchorHeight} =
+          anchor.content.getBoundingClientRect() ?? {};
+
+        // This checks whether or not the anchor has custom content with our own
+        // div wrapper. If not, that means we have a regular AdvancedMarker without any children.
+        // In that case we do not want to adjust the infowindow since it is all handled correctly
+        // by the Google Maps API.
+        if (anchorWidth === 0 && anchorHeight === 0) {
+          // We can safely typecast here since we control that element and we know that
+          // it is a div
+          const anchorDomContent = anchor.content.firstChild as Element;
```

Comment:
This could be risky, I think `firstChild` could also be a text- or even a comment-node, using `Element.firstElementChild` should be fine here.